modin/engines/base/io/io.py

```diff
@@ -778,9 +770,10 @@ def to_sql(
 
     @classmethod
     def to_pickle(cls, obj, path, compression="infer", protocol=4):
-        """Pickle (serialize) object to file using Pandas.
-        For parameters description please refer to Pandas API.
+        """
+        Pickle (serialize) object to file using Pandas.
 
+        For parameters description please refer to Pandas API.
```

Comment:
when https://github.com/modin-project/modin/pull/2969 is merged you could apply a decorator from it which should insert the proper link (if you give it correct arguments 😄 )

Reply:
Yes, let's get https://github.com/modin-project/modin/pull/2969 merged first, added `TODO` for now.